### ISSUE TITLE
Query Results fails to load queries with unsupported data types, like objects

### ISSUE THREAD

**Issue by ianmartorell** [getredash/redash#2676]
### Issue Summary

I'm trying to use the Query Results data source, which works fine when querying results from a Postgres query, but it produces an error when using results from a Mongo query.

### Steps to Reproduce

1. Add the Mongo data source
2. Create a query using the Mongo data source
3. Add the Query Results data source
4. Create a query like the following using the Query Results data source:

```sql
SELECT *
FROM query_1
```

This throws the following error: 
```
Error running query: Error binding parameter 5 - probably unsupported type.
```
And here's the stack trace from the server:
```
InterfaceError: Error binding parameter 5 - probably unsupported type.
at create_table (/app/redash/query_runner/query_results.py:101)
at create_tables_from_query_ids (/app/redash/query_runner/query_results.py:76)
at run_query (/app/redash/query_runner/query_results.py:127)
at run (/app/redash/tasks/queries.py:449)
```

### Technical details:

* Redash Version: 4.0.1+b4038
* Browser/OS: Not relevant
* How did you install Redash: Running it on Kubernetes

**Comment by arikfr** [comment]
I updated the title to better describe the issue. One possible solution here is to do call `json.dumps` on fields of type `object` before loading into SQLite.

**Comment by susodapop** [comment]
This bug also affects the current version on SaaS: 431d6fa

I can make the QueryResults data source work if I remove the object / array fields from the MongoDB projection. This doesn't really fix the problem because I still can't visualize data from those fields.

**Comment by vikask008** [comment]
hey there, any solution on this. I am facing the same issue. Getting error " Error running query: Error binding parameter 5 - probably unsupported type. " while executing the query.
Redash version - 5.0.2

1. monog query -
{
"collection":"collection_name"
}
2. Query Result query -
select * from query_130

kindly help

**Comment by arikfr** [comment]
This was addressed in #3887 and will be part of the next release.
